log_outgoing_requests/admin.py

```diff
@@ -31,10 +39,11 @@ class OutgoingRequestsLogAdmin(admin.ModelAdmin):
         "response_ms",
         "timestamp",
     )
-    list_filter = ("method", "status_code", "hostname")
+    list_filter = ("method", "status_code", "hostname", "timestamp")
```

Comment:
```suggestion
    list_filter = ("method", "timestamp", "status_code", "hostname")
```
method is a small list of possible values, timestamp is also going to be a limited number of filter options.

There are many (but ultimately finite amount) possible status code, so this list of filter values can become large, and the `hostname` field is entirely dependent on how many external services are consumed so that's a list that can have a large amount of options -> this filter should go last.